### DISCUSSION TITLE
Slow fade animation when switching spaces

### DISCUSSION THREAD

**Discussion by hamadahiro** [asmvik/yabai#1947] [Q&A] [answered]
I have the "Reduce Motion" option turned on in the System Settings, which kills the swipe animation when switching spaces, but it seems that the fade in/out animation takes about the same amount of time—quite slow. I've tried to find a way to disable or speed up the animation, but it seems there's no solution so far in the latest versions of macOS. How do you deal with it? Has anyone found a solution or a workaround, or have we all just gotten used to what we currently have?

I'm on Sonoma 14.1.

**Comment by asmvik** [comment, marked as answer]
The latest versions of yabai fixes this on Sonoma. (SIP must be disabled and scripting addition installed).

**Comment by ekaj2** [comment]
Seems that this has returned on the latest Sonoma (14.5).

**Reply by zach-is-my-name** [reply]
Damn this discussion is much fresher than I assumed.  ~~How did you originally disable this with yabai?~~ I've applied the latest os update and have managed to void the transition effect completely by doing the full install and config of Yabai and enabling the separate contexts App.  Yabai probably isn't even working for me and definitely isn't working with skhd. But somehow the nutshell that all of its config prep cracked open made way for Contexts to switch apps with same respoonse time as Linux
